flink-runtime/src/main/java/org/apache/flink/runtime/jobgraph/DistributionPattern.java

```diff
@@ -18,29 +18,15 @@
 
 package org.apache.flink.runtime.jobgraph;
 
-import org.apache.flink.runtime.executiongraph.ExecutionVertex;
-
 /**
  * A distribution pattern determines, which sub tasks of a producing task are connected to which
  * consuming sub tasks.
```

Comment:
Maybe add a comment like "It affects how execution edges are built in {@link EdgeManagerBuildUtil}."